lib/nerdtree/creator.vim

```diff
@@ -110,11 +109,6 @@ endfunction
 " FUNCTION: s:Creator._createNERDTree(path) {{{1
 function! s:Creator._createNERDTree(path, type)
     let b:NERDTree = g:NERDTree.New(a:path, a:type)
-    "TODO: This is kept for compatability only since many things use
-    "b:NERDTreeRoot instead of the new NERDTree.root
-    "Remove this one day
-    let b:NERDTreeRoot = b:NERDTree.root
```

Comment:
Hi @lifecrisis 
This change is causing Projectionist plugin to raise errors at startup. 
https://github.com/tpope/vim-projectionist/issues/89

I just want to highlight that but this is indeed something to be fixed in Projectionist

Reply:
Yup, we'll add that back. It's been a long time, so others should also be encouraged to update their code.